tests/php/BackupCode/RegisterHandlerTest.php

```diff
@@ -15,7 +16,7 @@
 
 class RegisterHandlerTest extends SapphireTest
 {
-    protected static $fixture_file = 'LoginHandlerTest.yml';
+    protected static $fixture_file = 'RegisterHandlerTest.yml';
```

Comment:
These changes aren't related to Login -> Verify, is this what's causing the builds to fail?

Reply:
Na that's not the build failure cause. Also I agree with the change (and the removal below of the code that worked around this problem). Aside from not really matching the commit message - it's all just minor test changes here anyway.

Reply:
No, these changes are reflecting that the test class was using the wrong fixture all along (and the correct one did exist and was not referenced by any other test class).

The test failures are an unfortunate race condition. It's not the first time I've seen it.